docs/omero_figure_scripting.rst

```diff
@@ -109,9 +121,30 @@ We will use the time-lapse images listed above to create a FRAP figure but you c
 
 #.  The labels should be added. Note that you can undo and redo these changes in the UI as normal.
 
-#.  Try out other JavaScript examples in the `scripts <https://github.com/ome/omero-guide-figure/tree/master/scripts>`_ folder.
-
 
 .. |script_icon| image:: images/scripts_icon.png
    :width: 0.36621in
    :height: 0.27231in
+
+
+Example 2: Shapes heatmap from OMERO.table
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This example uses an OMERO.table linked to each Image to generate
+a heatmap of colors applied to Shapes on the figure panel.
+Selected panels need to have Shapes added from OMERO (in the ROIs dialog).
+This means that each shape JSON will have an ``id`` that corresponds to a Shape in
```

Comment:
```suggestion
Thus, each shape JSON will have an ``id`` that corresponds to a Shape in
```

Maybe like this ? This is, I am assuming that the ids in JSON will be populated by virtue of adding the ROIs from OMERO ? If not, how does the JSON get the shapes ids ?